### PR TITLE
updated glossary based on discord feedback

### PR DESCRIPTION
**Pull request by xh3b4sd**

### CODE REVIEW

**Comment by johnnyonline** [comment]
**resources/glossary.md**
- "#### Target Asset" should maybe be "#### Concentrator Target Asset"
- "#### Vault Asset" - "The tokens that users deposit into a vault." should be more like "The primary vault asset, which shares are denominated in, and users deposits are swapped into."


**resources/smart-contracts/glp-compounder.md**
- Strategy section should be more like "vault stakes GLP, claims and stakes all the earned esGMX and multiplier points. The earned esGMX is never vested but instead used to boost earnings for more native tokens. The earned multiplier points also boost earnings for more native tokens. Vault then periodically claims rewards, swaps to some asset contained in the GLP and chosen by the harvester, and mints additional GLP to also earn more fees."

File: resources/glossary.md [hunk 1]
```diff
@@ -8,14 +8,18 @@ A vault re-investing generated rewards into itself.
...
+The tokens that concentrator rewards are swapped into.
 
-#### Underlying Asset
+#### Vault
+
+A smart contract managing token deposits according to its specific investment strategy.
+
+#### Vault Asset
 
 The tokens that users deposit into a vault.
```
Comment (xh3b4sd) [inline resources/glossary.md]:
```suggestion
The primary asset which vault shares are denominated in.
```

**Comment by xh3b4sd** [comment]
Please check again @johnnyonline. Tried to incorporate changes as discussed. Let me know if I got something wrong. Feel free to approve when all looks good.

File: resources/smart-contracts/glp-compounder.md [hunk 2]
```diff
@@ -5,4 +5,10 @@
 * Address: [https://arbiscan.io/address/0x9363e5e4a7aDfB346BEA0fe87a8BD02fddA48855](https://arbiscan.io/address/0x9363e5e4a7aDfB346BEA0fe87a8BD02fddA48855)
 * Vault Asset: GLP
 * Vault Shares: fortGLP
-* Strategy: GLP is being staked in GMX, earning more GLP from GMX traders and swappers.
+* Strategy: GLP is being staked in GMX, earning more esGMX and multiplier points
+  from GMX traders and swappers. The earned esGMX is never vested but instead
```
Comment (johnnyonline) [inline resources/smart-contracts/glp-compounder.md]: "GLP is being staked in GMX, earning fees from traders and token incentives from GMX."

Comment (johnnyonline) [reply resources/smart-contracts/glp-compounder.md]: other than that lgtm.

**Pull request merged**